packages/core/getUserAvailability.ts

```diff
@@ -343,7 +344,26 @@ const _getUserAvailability = async function getUsersWorkingHoursLifeTheUniverseA
     end: dayjs(busy.end),
   }));
 
-  const dateRangesInWhichUserIsAvailable = subtract(dateRanges, formattedBusyTimes);
+  const datesOutOfOffice = await getOutOfOfficeDays({
+    userId: user.id,
+    dateFrom,
+    dateTo,
+    availability,
+  });
+
+  // If the user is OOO with a forward to a member that day should be displayed as available
```

Comment:
My overall take it's deliver the right data from the backend and avoid post processing on the frontend.

Reply:
In this file we fetch out of office entries from the DB and parse them accordingly so slots and availability functions can use them normally.